src/main/resources/db/migration/V0_0_1_20240329124018__insert-oidc-registered-client.sql

```diff
@@ -7,7 +7,7 @@ scopes,
 client_settings,
 token_settings)
 VALUES('1d0ae4d0-0ec5-4f64-92aa-db46d718ebae', 'oidc-client', '2024-03-28 15:20:33',
```

Comment:
Please help me to add this credentail information to README file as well:
Default registered client infor:
- client_id: oidc-client
- client_secret: secret